base_user_role/models/group.py

```diff
@@ -0,0 +1,77 @@
+from odoo import api, fields, models
+
+
+class ResGroups(models.Model):
+    _inherit = "res.groups"
+
+    role_id = fields.One2many(
+        comodel_name="res.users.role",
+        inverse_name="group_id",
+        help="Relation for the groups that are associated with a role",
```

Comment:
```suggestion
        help="Relation for the groups that represents a role",
```
Because "associated with" could mean a group which is part of a role? A comment above saying that this field should be used as a "One2one" relation (that's why it is named like a Many2one I suppose).

Reply:
Thanks for the suggestion! I've applied the changes according to what you were commenting. Hope that works now! :smile: